integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/OpenshiftWithApplicationPropertiesTest.java

```diff
@@ -66,6 +67,14 @@ public void assertGeneratedResources() throws IOException {
             });
         });
 
-        assertThat(openshiftList).filteredOn(h -> "Route".equals(h.getKind())).hasSize(1);
+        assertThat(openshiftList).filteredOn(i -> "Route".equals(i.getKind())).hasOnlyOneElementSatisfying(i -> {
+            assertThat(i).isInstanceOfSatisfying(Route.class, in -> {
+                //Check that lables are also applied to Routes (#10260)
+                assertThat(i.getMetadata()).satisfies(m -> {
+                    assertThat(m.getName()).isEqualTo("test-it");
+                    assertThat(m.getLabels()).contains(entry("foo", "bar"));
```

Comment:
Can we also add a check for annotations?

Reply:
@iocanel can you please add a check for annotations as well?

Reply:
Sure